pkg/storage/secret/namespace.go

```diff
@@ -0,0 +1,14 @@
+package secret
+
+// Namespace is a newtype of string that improves type safety.
+type Namespace string
+
+func (n Namespace) String() string {
+	return string(n)
+}
+
+// NamespacedName is a tuple of name and namespace, often used by K8s resources for uniqueness.
+type NamespacedName struct {
```

Comment:
not to change, can this maybe be just NameNamespace? this sounds like Name is namespaced (if that's even a thing) 🤔

Reply:
Updated!